src/Core/Services/PolicyService.cs

```diff
@@ -180,6 +181,23 @@ public async Task Clear(string userId)
             return true;
         }
 
+        public Tuple<ResetPasswordPolicyOptions, bool> GetResetPasswordPolicyOptions(IEnumerable<Policy> policies,
+            string orgId)
+        {
+            var resetPasswordPolicyOptions = new ResetPasswordPolicyOptions();
+
+            if (policies == null || orgId == null)
+            {
+                return new Tuple<ResetPasswordPolicyOptions, bool>(resetPasswordPolicyOptions, false);
+            }
+
+            var policy = policies.FirstOrDefault(p =>
+                p.OrganizationId == orgId && p.Type == PolicyType.ResetPassword && p.Enabled);
+            resetPasswordPolicyOptions.AutoEnrollEnabled = GetPolicyBool(policy, "autoEnrollEnabled") ?? false;
+
+            return new Tuple<ResetPasswordPolicyOptions, bool>(resetPasswordPolicyOptions, policy?.Enabled ?? false);
```

Comment:
```suggestion
            return new Tuple<ResetPasswordPolicyOptions, bool>(resetPasswordPolicyOptions, policy != null);
```

The policy is `null` if not `Enabled`, so this check is redundant or at the least a little confusing. I would recommend just checking if the `policy` is `null` since the filter on `Enabled` was done when fetching the policy originally.

Reply:
Slightly redundant, but very explicit! Hah. Regardless, good suggestion. Thanks!